app/views/credit_cards/edit.html.erb

```diff
@@ -0,0 +1,8 @@
+<div class="container py-4">
+  <h3>Update Credit Card Information</h3>
+  <% if user_signed_in? %>
+    <%= render "form", credit_card: @credit_card %>
```

Comment:
Does this form enable you to pull current credit card values as place holders? think of what I did for reservations